recipe/meta.yaml

```diff
@@ -15,12 +15,7 @@ build:
   number: 0
   # arrow-cpp is not available for s390x, see arrow-cpp-feedstock for details.
   skip: true  # [s390x or py<37]
-  entry_points:
-    - plasma_store = pyarrow:_plasma_store_entry_point
```

Comment:
entry_points still exist, see https://github.com/apache/arrow/blob/a6eabc2b890030578131aecc5e85900597d694a4/python/setup.py#L748-L751. or is it not working?

Reply:
Thanks!